app/grandchallenge/products/templates/products/about.html

```diff
@@ -24,41 +24,175 @@ <h2 class="h5">Want to cite this website?</h2>
+          </div>
+        </div>
+
+        <div class="row pt-2 bg-light">
+          <div class="col-sm d-flex justify-content-between" data-toggle="collapse" href="#order-list" role="button" aria-expanded="false" aria-controls="order-list">
+            <p>What determines the order of the listing?</p>
+            <p><i class="fas fa-angle-down text-radboud"></i></p>
+          </div>
+        </div>
+
+        <div class="row pt-2">
+          <div class="col-sm collapse multi-collapse" id="order-list">
+            <p>The general order on the <a class="text-radboud" target="_blank" href="{% url 'products:product-list' %}">Products page</a> is alphabetically based on the product names. Products with MDR certification are listed first, followed by products for which the information was provided by the vendor (<i>Information source: Vendor</i>) and certification was verified (<i>Certification verified: Yes</i>).</p>
+          </div>
+        </div>
+
+        <div class="row pt-2 bg-light">
+          <div class="col-sm d-flex justify-content-between" data-toggle="collapse" href="#fund-1" role="button" aria-expanded="false" aria-controls="fund-1">
+            <p>Do companies pay to be listed?</p>
+            <p><i class="fas fa-angle-down text-radboud"></i></p>
+          </div>
+        </div>
+
+        <div class="row pt-2">
+          <div class="col-sm collapse multi-collapse" id="fund-1">
+            <p>No, companies do not pay to be listed in this overview of artificial intelligence software in radiology. We aim to increase transparency in the field and be as comprehensive as possible. This also means companies cannot pay to be prioritized in the listing order.</p>
+          </div>
+        </div>
+
+        <div class="row pt-2 bg-light">
+          <div class="col-sm d-flex justify-content-between" data-toggle="collapse" href="#fund-2" role="button" aria-expanded="false" aria-controls="fund-2">
+            <p>How is the platform funded?</p>
+            <p><i class="fas fa-angle-down text-radboud"></i></p>
+          </div>
+        </div>
+
+        <div class="row pt-2">
+          <div class="col-sm collapse multi-collapse" id="fund-2">
+            <p>The platform is currently funded as part of a research project by Radboud University Medical Center in Nijmegen, the Netherlands.</p>
```

Comment:
There's a bit of an inconsistency in the capitalization of `Radboud University Medical Center`, here you use title case, whereas earlier you only capitalized `Radboud`.

Reply:
good point! will quickly adapt that and push again. thanks

Reply:
I get a `fatal: unable to access could not resolve host github.com` notification when trying to push. Could this be because you already approved the pull request? Should I make a new branch for this?

Reply:
No you should be able to push. Are you on the vpn maybe?

Reply:
The 'try-turning-it-off-and-on-again' was the magic trick once again ;)